packages/webdriverio/src/commands/browser/throttle.js

```diff
@@ -1,12 +1,15 @@
 /**
  * Throttle the network capabilities of the browser. This can help to
- * emulate certain scenarios where a user looses the internet connection
+ * emulate certain scenarios where a user loses their internet connection
  * and your app needs to address that.
  *
+ * [There are many presets available](https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/src/commands/browser/throttle.js#L29)
```

Comment:
Shall we just list the presets rather than providing a link that might become outdated soon.

Reply:
I was thinking about that but was too lazy to do it :)

Reply:
@klamping Are you still up for updating it?

Reply:
Done :)